scripting/2.0/modules/ROOT/pages/scripting-module-studio.adoc

```diff
@@ -206,14 +205,12 @@ result = factorial(initialValue + int(payload))</scripting:code>
 
 === Target Variable and Target Value
 
-[%header,cols="20s,20a,35a,20a,5a"]
-|===
-| Name | Type | Description | Default Value | Required
-| Target Variable | String | The name of a variable on which to place the operation’s output | |
-| Target Value | String | An expression to evaluate against the operation’s output. The outcome of that expression will be stored in the target variable. | #[payload]|
-|===
+Use *Target Variable* parameter to define the name of a variable on which to place the operation’s output, and the *Target Value* to define an expression to evaluate against the operation’s output. The outcome of that expression will be stored in the target variable.
+
+To configure these parameters for the Scripting Module, follow the next steps:
```

Comment:
Same comment about not needing "follow the next steps." If you want to keep it in, change it to "follow these steps:"